app/supported-dexes/page.mdx

```diff
@@ -3,17 +3,12 @@ title: Supported DEXes
 description: List of supported DEXes and chains.
 ---
 
+import { RemoteMarkdown } from "../../components/remote-markdown";
+
 ## Supported DEXes
 
 Metrom is committed to support as many DEXes on as many chains as possible. If
 you want to see your DEX, AMM or network supported, reach out to us on
 [Discord](https://discord.gg/S2kBEAGWbM) and let's get you set up.
 
-| DEX                                   | [Scroll](https://scroll.io)                  | [Taiko](https://taiko.xyz)                   | [Base](https://www.base.org)                 | [Mode](https://www.mode.network)             | [Mantle](https://www.mantle.xyz)             |
-| ------------------------------------- | -------------------------------------------- | -------------------------------------------- | -------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
-| [Uniswap v3](https://app.uniswap.org) | <span className="greenText">Supported</span> | <span className="greenText">Supported</span> | -                                            | -                                            | -                                            |
-| [Panko](https://panko.finance)        | -                                            | <span className="greenText">Supported</span> | -                                            | -                                            | -                                            |
-| [Kim.exchange](https://kim.exchange)  | -                                            | -                                            | <span className="greenText">Supported</span> | <span className="greenText">Supported</span> | -                                            |
-| [Swapsicle](https://app.swapsicle.io) | -                                            | -                                            | -                                            | -                                            | <span className="greenText">Supported</span> |
-| [Scribe](https://app.scribe.exchange) | <span className="greenText">Supported</span> | -                                            | -                                            | -                                            | -                                            |
-| [BaseSwap](https://baseswap.fi)       | -                                            | -                                            | <span className="greenText">Supported</span> | -                                            | -                                            |
+<RemoteMarkdown url="https://raw.githubusercontent.com/metrom-xyz/monorepo/refs/heads/develop/packages/frontend/SUPPORTED_PROTOCOLS.md" />
```

Comment:
Wondering if it's ok to fetch the file from `develop`, not sure, maybe it's fine.